subprojects/android-test/rx-idler/src/main/kotlin/com/avito/android/test/rxidler/RxIdlingRegistry.kt

```diff
@@ -0,0 +1,110 @@
+package com.avito.android.test.rxidler
+
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.espresso.IdlingResource
+import com.squareup.rx3.idler.Rx3Idler
+import io.reactivex.rxjava3.core.Scheduler
+import io.reactivex.rxjava3.functions.Function
+import io.reactivex.rxjava3.functions.Supplier
+import io.reactivex.rxjava3.plugins.RxJavaPlugins
+
+/**
+ * Motivation https://github.com/square/RxIdler/issues/34
```

Comment:
It'd good to describe briefly the main idea here.

Reply:
Done